graph.go

```diff
@@ -95,7 +95,9 @@ type NodeRemover interface {
 // EdgeSetter is an interface for adding edges to a graph.
 type EdgeSetter interface {
 	// SetEdge adds an edge from one node to another.
-	// If the nodes do not exist, they are added.
+	// If the graph supports node addition the nodes
+	// will be added if they do not exist, otherwise
```

Comment:
> The use of "will" is based on the fact that the docs are written for the user of the type more than the implementor

Ok, then it makes sense. I was looking at it from the viewpoint of an implementor. Now my mind will be at peace.